### PR TITLE
Adds item-page, explore, codelab and course examples to visual test suite 

### PR DESCRIPTION
**Pull request by johnrich85**
Fixes: https://github.com/GoogleChrome/web.dev/issues/9369

Changes proposed in this pull request:

- Adds item-page, export, codelab and course examples to handbook/content-types
- Uses the new pages for visual testing.

### CODE REVIEW

File: src/lib/utils/path.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
```
Comment (matthiasrohmer) [inline src/lib/utils/path.js:1]: See https://github.com/GoogleChrome/web.dev/pull/9413#pullrequestreview-1263713377 for details.

File: src/site/_data/courses/example/meta.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# When true the course will not be listed on the /learn page
+invisible: true
+title: i18n.courses.example.title
+description: i18n.courses.example.description
+# This is the logo that appears in the navigation drawer and header.
+logo: '/images/courses/accessibility/logo.svg'
+# This is used to provide the background for the navigation drawer and header.
+background: '/images/courses/accessibility/background.svg'
+# This provides the cover image for the card on /learn/
+card: '/images/courses/accessibility/card.svg'
```
Comment (matthiasrohmer) [inline src/site/_data/courses/example/meta.yml:19]: I wonder if we should use specific example assets here instead? Now whenever (if ever) the a11y assets change, our test page would also change.

File: src/site/_data/i18n/courses.yml [hunk 3]
```diff
@@ -92,3 +92,9 @@ learn_html:
     en: Learn HTML!
   description:
     en: "This HTML course for web developers provides a solid overview for developers, from novice to expert level HTML."
+
+example:
```
Comment (matthiasrohmer) [inline src/site/_data/i18n/courses.yml:96]: Did you verify this doesn't pop up anywhere where it should not? Algolia index, pages where courses are listed, ...?

File: src/site/_includes/learn-page.njk [hunk 4]
```diff
@@ -25,7 +25,7 @@ pageScripts:
     <div class="wrapper auto-grid learn__collection">
       {% set coursesArr = helpers.values(courses) %}
       {% for course in coursesArr|sort(true, false, 'meta.date') %}
-      {% if course.meta and not course.meta.draft %}
+      {% if course.meta and not course.meta.draft and not course.meta.invisible %}
```
Comment (matthiasrohmer) [inline src/site/_includes/learn-page.njk:28]: Please double-check that condition is not required somewhere else as well.

Comment (johnrich85) [reply src/site/_includes/learn-page.njk:28]: Had a look around and LGTM. Suspects were: tags, algolia and the courses page & it seems like the exclusions are working nicely.

**Review by matthiasrohmer — APPROVED** [review]

**Comment by johnrich85** [comment]
Hey @rachelandrew . Would you mind giving the new sample content a once over, please?

File: src/site/content/en/handbook/content-types/example-codelab/index.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+layout: codelab
+title: Example codelab
+authors:
+  - adamargyle
+description: >-
+  This page is used for visual testing of the codelab layout and also
+  showcases it's capabilities
```
Comment (rachelandrew) [inline src/site/content/en/handbook/content-types/example-codelab/index.md:8]:
```suggestion
  showcases its capabilities
```

File: src/site/content/en/handbook/content-types/example-course/example-section-2/index.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+---
+title: Example Section - 2
+authors:
+  - cariefisher
+description: This page is used for visual testing of the courses layout and also showcases it's capabilities
```
Comment (rachelandrew) [inline src/site/content/en/handbook/content-types/example-course/example-section-2/index.md:5]:
```suggestion
description: This page is used for visual testing of the courses layout and also showcases its capabilities
```

File: src/site/content/en/handbook/content-types/example-course/example-section/index.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+---
+title: Example Section
+authors:
+  - cariefisher
+description: This page is used for visual testing of the courses layout and also showcases it's capabilities
```
Comment (rachelandrew) [inline src/site/content/en/handbook/content-types/example-course/example-section/index.md:5]:
```suggestion
description: This page is used for visual testing of the courses layout and also showcases its capabilities
```

File: src/site/content/en/handbook/content-types/example-course/index.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
+---
+title: Example course
+authors:
+  - cariefisher
+description: This page is used for visual testing of the courses layout and also showcases it's capabilities
```
Comment (rachelandrew) [inline src/site/content/en/handbook/content-types/example-course/index.md:5]:
```suggestion
description: This page is used for visual testing of the courses layout and also showcases its capabilities
```

File: src/site/content/en/handbook/content-types/example-course/index.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+Adding courses involves the following:
+
+- Add a new directory to `/src/site/content/en/learn`
+- Add an index.md file. You can copy this one from `src/site/content/en/handbook/content-types/example-course/index.md` as a starting point
+- Add a data file named using the following convention: `{dir_name}.11tydata.js`
+- Use the `course11tydata` util to generate the required data. You can use the following file as a starting point `src/site/content/en/handbook/content-types/example-course/example-course.11tydata.js`
+- Create a new directory under `src/site/_data/courses` using the name passed to the `course11tydata` util.
+- Add two files, `meta.yml` and `toc.yml`.
+- Add course sections as subdirectories
+- Add the section urls to `toc.yml` and the sidebar will be generated automatically.
```
Comment (rachelandrew) [inline src/site/content/en/handbook/content-types/example-course/index.md:29]:
```suggestion
- Add a new directory to `/src/site/content/en/learn`.
- Add an index.md file. You can copy this one from `src/site/content/en/handbook/content-types/example-course/index.md` as a starting point.
- Add a data file named using the following convention: `{dir_name}.11tydata.js`.
- Use the `course11tydata` util to generate the required data. You can use the following file as a starting point `src/site/content/en/handbook/content-types/example-course/example-course.11tydata.js`.
- Create a new directory under `src/site/_data/courses` using the name passed to the `course11tydata` util.
- Add two files, `meta.yml` and `toc.yml`.
- Add course sections as subdirectories.
- Add the section urls to `toc.yml` and the sidebar will be generated automatically.
```

File: src/site/content/en/handbook/content-types/example-item-page/index.md [hunk 10]
```diff
@@ -0,0 +1,122 @@
+---
+layout: item-page
+path: authors
+showHero: true
+imageType: 'round'
+title: Authors
+renderData:
+  title: Sample item page
+  description: This page is used for visual testing of the item-page layout and also showcases it's capabilities
```
Comment (rachelandrew) [inline src/site/content/en/handbook/content-types/example-item-page/index.md:9]:
```suggestion
  description: This page is used for visual testing of the item-page layout and also showcases its capabilities
```

**Review by rachelandrew — APPROVED** [review]
Please commit my copyedits before merging.

**Pull request merged**